NEWS.md

```diff
@@ -1,7 +1,9 @@
 # mmrm 0.2.2.9001
 
 ### New Features
+
 - Add Kenward-Roger support for spatial covariance structures.
+- Add optional tidymodels support (#214, @dgkf)
```

Comment:
Sure, will make the change. 

In general, I think this is a great way of showing traceability of changes in case there are downstream effects or if users are curious about the decision-making process. The authorship bit to me is less critical, and not as widespread, but it is sort of a nice way to annotate code/feature ownership and elevate community contributors. 

It's seems to be quite a common practice among `r-lib` packages. Just picking the first three packages in my history: [remotes](https://github.com/r-lib/remotes/blob/main/NEWS.md), [covr](https://github.com/r-lib/covr/blob/main/NEWS.md), [dplyr](https://github.com/tidyverse/dplyr/blob/main/NEWS.md)